bin/install.sh

```diff
@@ -82,19 +133,25 @@ if [ ! -f /etc/locale.gen ]; then
   sudo locale-gen
 fi
 
-
 sudo sed -i 's/apt.screenlyapp.com/archive.raspbian.org/g' /etc/apt/sources.list
-sudo apt-get update
+sudo apt update -y
```

Comment:
`apt` will break in older versions of Debian (must use `apt-get`). I'm not sure how common this is, but I don't see much upside in using `apt` over `apt-get` (other than it is newer).

Reply:
I updated that line with apt-get.  That was there in the original code.  I did not notice it.